libobs/obs-nix.c

```diff
@@ -236,6 +236,34 @@ static void log_kernel_version(void)
 	blog(LOG_INFO, "Kernel Version: %s %s", info.sysname, info.release);
 }
 
+static void log_x_info(void)
+{
+	Display *dpy = XOpenDisplay(NULL);
+	if (!dpy) {
+		blog(LOG_INFO, "Unable to open X display");
+		return;
+	}
+
+	int        protocol_version  = ProtocolVersion (dpy);
+	int        protocol_revision = ProtocolRevision (dpy);
```

Comment:
The space after the function name and before the function parentheses isn't consistent with coding standards.  Make sure to watch out for this in the future.